### DISCUSSION TITLE
Show float values to only 2nd place of decimal in elastalert emails

### DISCUSSION THREAD

**Discussion by Singh-Shivani** [jertel/elastalert2#606] [Q&A] [answered]
Hi everyone,
How can I Limit a decimal value to only 2 decimal places in elastalert??

My rules.yaml file -

```
alert_subject: "Alerts: ErrorPercentage {0} against threshold {1}"
alert_subject_args:
- "ErrorPercentage"
- "Upper_Threshold"

```
### In emails I get -
ErrorPercentage - 0.44499381953028205
### In emails I want -
ErrorPercentage- 0.44

**Comment by jertel** [comment, marked as answer]
A similar question was asked earlier this year: #328

**Reply by Singh-Shivani** [reply]
Thank you! @jertel
